tools/webpack/packages.js

```diff
@@ -143,29 +152,103 @@ module.exports = {
 	performance: {
 		hints: false, // disable warnings about package sizes
 	},
+	optimization: {
+		...baseConfig.optimization,
+		runtimeChunk: {
```

Comment:
Does it make sense to enable the runtime chunk also on production? I was wondering whether it's something that we should also add to the `@wordpress/scripts` if there is more than a single chunk to ensure it works correctly with React Fast Refresh. However, my initial thinking was it only is useful in the dev mode when using hot module reloading.

Reply:
Generally, we want to use a single runtime chunk when we have multiple entries in webpack. The size of the runtime is minimal in production though, and it doesn't seem to impact the overall bundle sizes that much (see https://github.com/WordPress/gutenberg/pull/64444#issuecomment-2284090019). Extracting the runtime makes it easier to integrate with webpack features like HMR and code splitting though. I think it makes sense to do it in both development and production mode.